lib/daemon_runner/client.rb

```diff
@@ -24,6 +24,9 @@ def scheduler.on_error(job, error)
         mutex = @mutexes[task_id]
 
         logger.error "#{task_id}: #{error}"
+        error.backtrace.each do |l|
+          logger.error "#{task_id}: #{l}"
```

Comment:
Do you want to generate one log-event per backtrace line? I'd try to concatenate these and attach them to the initial `error` event or log them as one debug event.